python/pfs/drp/stella/reduceExposure.py

```diff
@@ -230,6 +231,28 @@ def run(self, sensorRefList):
         self.write(sensorRefList, results)
         return results
 
+    def runIsr(self, sensorRef):
+        """Run Instrument Signature Removal (ISR)
+
+        This method exists to allow us to post-process the ISR-ed image.
```

Comment:
How about 'This method wraps the ISR call to allow us to post-process the ISR-ed image.'